lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -738,33 +744,43 @@ sub _failed_check {
     # Count has to reflect the actual number of rows
     my $count = 0;
     my $hiddens = {table => $check->table,
-                    edit => $check->column,
-                           id_column => $check->{id_column},
-                            id_where => $check->{id_where},
+               id_column => $check->{id_column},
+                id_where => $check->{id_where},
+                  insert => $check->{insert},
                 database => $request->{database}};
+    my $i = 1;
+    # Move around Can't use string "ARRAY as an ARRAY ref while "strict refs" in use
```

Comment:
This comment is a complete mystery to me... Are you sure it's both still relevant and the best way to document the code below it?
(Note: to me, the code below looks pretty straight forward...)